doc/source/migration.rst

```diff
@@ -1,3 +1,79 @@
 =================
 Migration process
 =================
+
+  
+#. Clone docs-exports repository 
+
+    * To start the Migration Process you have to clone the Docs-Exports
+
+
+     This can be achived with the following command:
+  
+      .. code-block:: bash
+
+      git clone https://github.com/opentelekomcloud-docs/doc-exports
```

Comment:
now you need to add indentation for the code-block content

Reply:
I totally got the identation wrong initally but I think now it looks ok.